src/course-unit/hooks.jsx

```diff
@@ -40,13 +40,14 @@ export const useCourseUnit = ({ courseId, blockId }) => {
   const sequenceStatus = useSelector(getSequenceStatus);
   const { draftPreviewLink, publishedPreviewLink } = useSelector(getCourseSectionVertical);
   const courseVerticalChildren = useSelector(getCourseVerticalChildren);
-  const enableCopyPasteUnits = useSelector(getCourseUnitEnableCopyPaste);
   const staticFileNotices = useSelector(getStaticFileNotices);
   const navigate = useNavigate();
   const isEditTitleFormOpen = useSelector(state => state.courseUnit.isEditTitleFormOpen);
   const isQueryPending = useSelector(state => state.courseUnit.isQueryPending);
+  const canEdit = useSelector(getCanEdit);
```

Comment:
[question] Can we move declaration const `canEdit = useSelector(getCanEdit);` to the `useCopyToClipboard`? Because use are using it only there.

Reply:
Moved